### DISCUSSION TITLE
madmax RAM disk

### DISCUSSION THREAD

**Discussion by iamchriswick** [guydavis/machinaris#107] [Q&A] [answered]
Where do we set madmax RAM disk path?

**Comment by guydavis** [comment, marked as answer]
Hi, please use the 'directories => tmp2' setting in [plotman.yaml](https://github.com/guydavis/machinaris/wiki/MadMax).  This is passed to madmax chia_plot as `-2`.

**Reply by iamchriswick** [reply]
Thanks :+1:
